### PR TITLE
7 nc news

### PR DESCRIPTION
**Pull request by crossmatthew**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -160,16 +160,73 @@ describe('GET /api/articles/:article_id/comments', () => {
...
+        return request(app)
+        .post('/api/articles/1/comments')
+        .expect(400)
+        .send({
+            username: true,
+            body: 'aaaasss'
+        })
+        .then(({ body }) => {
+            expect(body.code).toBe('23503')
+            expect(body.msg).toBe('Bad Request')
```
Comment (MRombi) [inline __tests__/app.test.js:208]: We are missing a test for when the user is not a valid one (404)  🙂

File: models/comments-model.js [hunk 2]
```diff
@@ -16,4 +16,19 @@ exports.commentsOfArticle = (req) => {
...
         return data.rows
-    }})}
\ No newline at end of file
+    }})};
+exports.commentToPost = (req) => {
+    const { body, params } = req
+    if (!body) {
+        return Promise.reject()
+    }
+    return checkExists('articles', 'article_id', params.article_id)
```
Comment (MRombi) [inline models/comments-model.js]: We could use a promise.all in here 🙂

**Review by MRombi — CHANGES_REQUESTED** [review]
Good job on this endpoint! Please, action on the feedback I left and resubmit. Let me know if you have any question 🙂

**Review by aloush-dev — APPROVED** [review]
Nice job actioning feedback.

This is ready to merge.

**Pull request merged**